tests/php/sync/test_class.jetpack-sync-integration.php

```diff
@@ -2,6 +2,52 @@
 
 class WP_Test_Jetpack_Sync_Integration extends WP_Test_Jetpack_Sync_Base {
 
+	function test_get_sync_status() {
+		$no_checksum = Jetpack_Sync_Actions::get_sync_status();
+		$this->assertArrayNotHasKey( 'posts_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comments_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'post_meta_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $no_checksum );
+
+		$kitchen_sink_checksum = Jetpack_Sync_Actions::get_sync_status(
+			'posts_checksum,comments_checksum,post_meta_checksum,comment_meta_checksum'
+		);
+		$this->assertArrayHasKey( 'posts_checksum', $kitchen_sink_checksum );
+		$this->assertArrayHasKey( 'comments_checksum', $kitchen_sink_checksum );
+		$this->assertArrayHasKey( 'post_meta_checksum', $kitchen_sink_checksum );
+		$this->assertArrayHasKey( 'comment_meta_checksum', $kitchen_sink_checksum );
+
+		$posts = Jetpack_Sync_Actions::get_sync_status( 'posts_checksum' );
+		$this->assertArrayHasKey( 'posts_checksum', $posts );
+		$this->assertArrayNotHasKey( 'comments_checksum', $posts );
+		$this->assertArrayNotHasKey( 'post_meta_checksum', $posts );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $posts );
+
+		$comments = Jetpack_Sync_Actions::get_sync_status(
+			'comments_checksum'
+		);
+		$this->assertArrayNotHasKey( 'posts_checksum', $comments );
+		$this->assertArrayHasKey( 'comments_checksum', $comments );
+		$this->assertArrayNotHasKey( 'post_meta_checksum', $comments );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $comments );
+
+		$post_meta = Jetpack_Sync_Actions::get_sync_status(
+			'post_meta_checksum'
+		);
+		$this->assertArrayNotHasKey( 'posts_checksum', $post_meta );
+		$this->assertArrayNotHasKey( 'comments_checksum', $post_meta );
+		$this->assertArrayHasKey( 'post_meta_checksum', $post_meta );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $post_meta );
+
+		$comment_meta = Jetpack_Sync_Actions::get_sync_status(
```

Comment:
Should we add some test cases for other sync status keys, other than the checksum fields?

Reply:
No. In this case, it's just testing a specific concept and while the test is long, I just wish there was a simpler way to write it. :)